commands/publish/command.js

```diff
@@ -35,6 +35,11 @@ exports.builder = yargs => {
       type: "string",
       requiresArg: true,
     },
+    "legacy-auth": {
+      describe: "Legacy Base64 Encoded username and password.",
+      alias: "_auth",
```

Comment:
`--_auth` is an extremely odd option name. Please don't add this alias.

Reply:
This is actually the same as the npm `_auth` flag. So using that as a signal that it's a pass through.